project.xml

```diff
@@ -332,6 +333,9 @@
     <skip-file name="config.xml"/>
     </project>
     <!-- android_packages_wallpapers_Basic -->
```

Comment:
Please change android_packages_wallpapers_Basic to change android_packages_apps_VoiceDialer